sysutils/terraform/Portfile

```diff
@@ -25,8 +25,10 @@ homepage            https://www.terraform.io/downloads.html
 master_sites        https://releases.hashicorp.com/${name}/${version}
 distname            ${name}_${version}_darwin_amd64
 
-checksums           rmd160 4a7837ead34ea1c978b2cc817dac41869338eb97 \
-                    sha256 183078bf230e517e6f41e47d6e7d3b61093c6bb5a2b85958c01a4cf3949b7c14
+checksums           md5    4298313370faeedefa5695e4bd61c228 \
```

Comment:
Just an FYI, I don't think md5s are still desired in Portfiles. (rmd160, sha256, and size is I think the current standard). Don't bother removing it now, but when you update things in the future md5 isn't needed.

Reply:
Oh boy, I thought I was doing these ports a favour. Ok I will update my scripts and I'll remove all the `md5` checksums on the next PRs.